gpflow/kernels/mo_kernels.py

```diff
@@ -76,10 +85,11 @@ class SharedIndependent(MultioutputKernel):
     Use `gpflow.kernels` instead for more efficient code.
     """
     def __init__(self, kernel: Kernel, output_dimensionality: int):
```

Comment:
```suggestion
    def __init__(self, kernel: Kernel, output_dim: int):
```

Reply:
https://github.com/GPflow/GPflow/pull/1167#discussion_r353256138

Reply:
Where `kwargs` come from?

Reply:
from `__init__`

Reply:
Why init has `kwargs`? Why it has been changed in a combination kernel? It looks like a big change in a small PR hidden away.

Reply:
I think if we know that the this class (not abstract) will not be inherited from we might not need the kwargs. See my big comment below